android_app/app/src/main/res/values/strings.xml

```diff
@@ -70,55 +70,55 @@
+    <string name="error_initial_weight_required">Error: Initial weight required</string>
+    <string name="error_goal_weight_required">Error: Goal weight required</string>
+
+    <string name="info_data_deleted">Entry deleted</string>
+    <string name="info_data_all_deleted">All entries deleted</string>
+    <string name="info_data_exported">Exported to</string>
+    <string name="info_data_imported">Imported from</string>
+    <string name="info_enter_value_cm">Value in cm:</string>
+    <string name="info_enter_comment">Optional comment:</string>
+    <string name="info_enter_value_scale_unit">Value in scale unit:</string>
     <string name="info_is_visible">is visible</string>
     <string name="info_is_not_visible">is not visible</string>
-    <string name="info_is_enable">is enabled</string>
-    <string name="info_is_not_enable">is disabled</string>
-    <string name="info_is_not_available">is not available</string>
-    <string name="info_bluetooth_try_connection">Trying to connect to</string>
+    <string name="info_is_enable">enabled</string>
+    <string name="info_is_not_enable">disabled</string>
+    <string name="info_is_not_available">not available</string>
+    <string name="info_bluetooth_try_connection">Connecting to:</string>
     <string name="info_bluetooth_connection_lost">Lost Bluetooth connection</string>
     <string name="info_bluetooth_no_device">No Bluetooth device found</string>
     <string name="info_bluetooth_no_device_set">No Bluetooth device selected</string>
-    <string name="info_bluetooth_connection_successful">Connection successful established</string>
+    <string name="info_bluetooth_connection_successful">Connection established</string>
     <string name="info_bluetooth_init">Initialize Bluetooth device</string>
-    <string name="info_bluetooth_connection_error">Bluetooth has an unexpected error</string>
+    <string name="info_bluetooth_connection_error">Unexpected Bluetooth error</string>
     <string name="info_new_data_added">%1$.2f%2$s [%3$s] to %4$s added</string>
     <string name="info_new_data_duplicated">measurement with the same date and time already exist</string>
 
     <string name="info_enter_user_name">Enter your name</string>
-    <string name="info_no_selected_user">No user exists. Please create a new user in the settings.</string>
-    <string name="info_no_evaluation_available">Can\'t evaluate the value</string>
+    <string name="info_no_selected_user">No user exists. Please create one in the settings.</string>
+    <string name="info_no_evaluation_available">Could not evaluate the value</string>
 
-    <string name="question_really_delete">Do you really want to delete database entry?</string>
-    <string name="question_really_delete_all">Do you really want to delete all database entries?</string>
-    <string name="question_really_delete_user">Do you really want to delete the user? </string>
+    <string name="question_really_delete">Confirm deletion of entry.</string>
```

Comment:
The buttons in the dialog where this and the two below are shown are yes and no. Doesn't the previous text sound better?

Reply:
It is an unmarked noun error. Should be "the database entry". It is longer, in a more commonly used convention. I favor mine, but I can change it.